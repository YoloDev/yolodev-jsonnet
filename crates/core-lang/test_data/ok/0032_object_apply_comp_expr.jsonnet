local CCompiler = {};

CCompiler { [x]: true for x in ['1'] }
