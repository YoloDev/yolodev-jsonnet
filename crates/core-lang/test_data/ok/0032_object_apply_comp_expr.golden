local CCompiler#2 = {};
(CCompiler#2 + [
    local x#5 = $arr#4[0.0];
    x#5
  ]:
  local x#5 = $arr#4[0.0];
    true
  for $arr#4 in
  local $arr#6 = ["1"];
    std#1["join"]([],
      std#1["makeArray"](std#1["length"]($arr#6),
        function($i#7 = error "Parameter \'$i\' not bound")
          local x#5 = $arr#6[$i#7];
          [[x#5]],
      ),
    ))
